src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+import { ReactPropTypes } from 'react';
+import uuid from 'react-uuid';
+
+const Button = ({
+  className, value, onClick, id,
+}) => (
+  <button type="button" className={className} onClick={onClick} id={id}>
+    {value}
+  </button>
+);
+
+const buttons = [
+  ['AC', '+/-', '%', '÷'],
+  [7, 8, 9, 'x'],
+  [4, 5, 6, '-'],
+  [1, 2, 3, '+'],
+  [0, '.', '='],
+];
+
+Button.propTypes = {
+  className: ReactPropTypes.string.isRequired,
+  value: ReactPropTypes.string.isRequired,
+  onClick: ReactPropTypes.func.isRequired,
+  id: ReactPropTypes.string.isRequired,
+};
```

Comment:
- [ ] You did a great job refactoring your code, it certainly looks cleaner! 💯 However, this piece of code introduced a bug:

![image](https://user-images.githubusercontent.com/77069011/218870562-5175d554-f7c0-4749-a1dc-5e715fe9222f.png)

Please refer to this [article of the official React docs](https://reactjs.org/docs/typechecking-with-proptypes.html) to correct the way you are type-checking your props! 🤓 🔧

Reply:
Dear @anagudelogu, Thank you very much for the awesome review I have now done it properly but the onClick because I have not implemented it yet it doesn't return a function and so shows the error for that, should I implement it? Since the exercise is about making the calculator UI only.